Docker build fails

## How to reproduce

- Commit `b049ea3`, Docker version 27.1.2, build d01f264
- Run `docker build .` in the root dir
- Lament

```
...
13.69 error[E0282]: type annotations needed for `Box<_>`
13.69   --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.30/src/format_description/parse/mod.rs:83:9
13.69    |
13.69 83 |     let items = format_items
13.69    |         ^^^^^
13.69 ...
13.69 86 |     Ok(items.into())
13.69    |              ---- type must be known at this point
13.69    |
13.69 help: consider giving `items` an explicit type, where the placeholders `_` are specified
13.69    |
13.69 83 |     let items: Box<_> = format_items
13.69    |              ++++++++
13.69
13.70    Compiling httparse v1.8.0
13.90    Compiling crossbeam-utils v0.8.16
14.03    Compiling termcolor v1.4.0
14.16    Compiling ppv-lite86 v0.2.17
14.23    Compiling safemem v0.3.3
14.47    Compiling thiserror v1.0.50
14.91    Compiling libm v0.2.8
14.95    Compiling matrixmultiply v0.1.15
15.14    Compiling minimal-lexical v0.2.1
15.35    Compiling rand_chacha v0.3.1
15.44    Compiling http v0.2.11
15.58    Compiling line-wrap v0.1.1
15.76    Compiling nom v7.1.3
15.87    Compiling actix-service v2.0.2
15.92 For more information about this error, try `rustc --explain E0282`.
15.94 error: could not compile `time` (lib) due to 1 previous error
15.95 warning: build failed, waiting for other jobs to finish...
------
Dockerfile:6
--------------------
   4 |     COPY . /sources
   5 |     WORKDIR /sources
   6 | >>> RUN cargo build --release
   7 |     RUN chown nobody:nogroup /sources/target/release/bin
   8 |
--------------------
ERROR: failed to solve: process "/bin/sh -c cargo build --release" did not complete successfully: exit code: 101
s
```

> ## How to reproduce
> 
> - Commit `b049ea3`, Docker version 27.1.2, build d01f264
> - Run `docker build .` in the root dir
> - Lament
> 
> ```
> ...
> 13.69 error[E0282]: type annotations needed for `Box<_>`
> 13.69   --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.30/src/format_description/parse/mod.rs:83:9
> 13.69    |
> 13.69 83 |     let items = format_items
> 13.69    |         ^^^^^
> 13.69 ...
> 13.69 86 |     Ok(items.into())
> 13.69    |              ---- type must be known at this point
> 13.69    |
> 13.69 help: consider giving `items` an explicit type, where the placeholders `_` are specified
> 13.69    |
> 13.69 83 |     let items: Box<_> = format_items
> 13.69    |              ++++++++
> 13.69
> 13.70    Compiling httparse v1.8.0
> 13.90    Compiling crossbeam-utils v0.8.16
> 14.03    Compiling termcolor v1.4.0
> 14.16    Compiling ppv-lite86 v0.2.17
> 14.23    Compiling safemem v0.3.3
> 14.47    Compiling thiserror v1.0.50
> 14.91    Compiling libm v0.2.8
> 14.95    Compiling matrixmultiply v0.1.15
> 15.14    Compiling minimal-lexical v0.2.1
> 15.35    Compiling rand_chacha v0.3.1
> 15.44    Compiling http v0.2.11
> 15.58    Compiling line-wrap v0.1.1
> 15.76    Compiling nom v7.1.3
> 15.87    Compiling actix-service v2.0.2
> 15.92 For more information about this error, try `rustc --explain E0282`.
> 15.94 error: could not compile `time` (lib) due to 1 previous error
> 15.95 warning: build failed, waiting for other jobs to finish...
> ------
> Dockerfile:6
> --------------------
>    4 |     COPY . /sources
>    5 |     WORKDIR /sources
>    6 | >>> RUN cargo build --release
>    7 |     RUN chown nobody:nogroup /sources/target/release/bin
>    8 |
> --------------------
> ERROR: failed to solve: process "/bin/sh -c cargo build --release" did not complete successfully: exit code: 101
> s
> ```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,6 +26,7 @@ bat = "0.24"
 syntect = "5.1"
 tokio = { version = "1.35", features = ["sync"] }
 futures = "0.3"
+time = "0.3.36"
 
 [profile.release]
 lto = true

